src/CoreBundle/private/library/classes/dbobjects/TCMSImageEndpoint.class.php

```diff
@@ -733,15 +733,11 @@ public static function imagecopymerge_alpha($dst_im, $src_im, $dst_x, $dst_y, $s
      */
     public function GetRelativeURL()
     {
-        /** @var string|null $dataPath */
-        $dataPath = $this->aData['path'] ?? '';
-
-        if (false === $this->_isThumbnail) {
-            $sImageURL = $this->GetImageUrlPathPrefix().$dataPath;
-        }else{
-            $sImageURL = URL_MEDIA_LIBRARY_THUMBS_PATH.$this->GetThumbPathExtension().'/'.$dataPath;
+        if (!$this->_isThumbnail) {
```

Comment:
Our current code convention is false ===

Reply:
it was just rollback. But, its updated.